product_plm_import/wizards/product_plm_import.py

```diff
@@ -69,6 +70,21 @@ def _update_vals_list(self, row_dict, error_list, vals_list):
         product = self.env["product.product"].search(product_domain)
         if product:
             error_list.append(_("There is already a product for %s.", part_number))
+        quality_check_categ_code = row_dict.get("quality_check_categ_id")
+        if quality_check_categ_code:
+            quality_check_categ = self.env["quality.check.category"].search(
+                [("code", "=", quality_check_categ_code)], limit=1
+            )
+            row_dict["quality_check_categ_id"] = (
+                quality_check_categ.id if quality_check_categ else False
+            )
+            if not quality_check_categ:
+                error_list.append(
+                    _(
+                        "A non-existent inspection category (%s) is present in the I/F file."
+                    )
+                    % quality_check_categ_code
+                )
```

Comment:
```suggestion
            row_dict["quality_check_categ_id"] = quality_check_categ.id
            if not quality_check_categ:
                error_list.append(
                    _("Quality Check Category (%s) does not exist.", quality_check_categ_code)
                )
```

See this: https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#use-translation-method-correctly